include/seqan3/alphabet/composition/cartesian_composition.hpp

```diff
@@ -171,18 +171,18 @@ decltype(auto) get();
 
 /*!\brief The CRTP base for a combined alphabet that contains multiple values of different alphabets at the same time.
  * \ingroup composition
- * \implements seqan3::semi_alphabet_concept
- * \implements seqan3::detail::constexpr_semi_alphabet_concept
- * \tparam first_component_type Type of the first letter; must model seqan3::semi_alphabet_concept.
- * \tparam component_types      Types of further letters (up to 4); must model seqan3::semi_alphabet_concept.
+ * \implements seqan3::SemiAlphabet
+ * \implements seqan3::detail::ConstexprSemiAlphabet
+ * \tparam first_component_type Type of the first letter; must model seqan3::SemiAlphabet.
```

Comment:
We re-discussed this and we decided on renaming `semi_alphabet_concept` to `Semialphabet` not `SemiAlphabet` (and for Constexpr, also).

Reply:
(English language peculiarities)